kieserver/modules/kieserver/module.yaml

```diff
@@ -26,6 +26,9 @@ artifacts:
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
+- name: "jbpm-event-emitters-kafka-7.x.Final.jar"
+  # jbpm-event-emitters-kafka-7.52.0.Final-redhat-00004.jar
+  md5: "07e2ec53eb779fab36b357b145ba255b"
```

Comment:
For RHDM image, these changes does not make sense, at it's jbpm-evnet-emitter. Changes are required only for RHPAM Kie Server image as RHDM Kie Server image do not have BPM capability (not possible to run jbpm)

Reply:
ok